### DISCUSSION TITLE
Is there a threshold to how minimal contributions can be?

### DISCUSSION THREAD

**Discussion by nalka0** [dotnet/maui#14250] [Q&A] [answered]
I'll take my actual case as an example but the best would be a more general answer.

I have spotted a [typo in the docs](https://github.com/dotnet/maui/blob/60c2b91c7c367bea29bc1ab3e0591a554ddadb42/src/Controls/docs/Microsoft.Maui.Controls/ResourceDictionary.xml#L144) ("Adds styleSheet tHo this..."). 

- Is it worth creating a PR that removes the typo?
- Is it worth creating an issue?
- Both?
- Do nothing and just wait for someone to remove it while doing something else?

I couldn't find something on this topic in [CONTRIBUTING.md](https://github.com/dotnet/maui/blob/60c2b91c7c367bea29bc1ab3e0591a554ddadb42/.github/CONTRIBUTING.md)

**Comment by hartez** [comment, marked as answer]
You can absolutely create a PR to fix this, and it will be much appreciated - I _hate_ when I see typos in our docs. 

Issues are also fine, though it might be a while before someone on the dev team can get around to a fix for this. 

Generally speaking, no issue or PR is too small, as long as it addresses something that's clearly incorrect (like the typo you pointed out). Just please don't submit PRs or issues for matters of taste (like brace placement), whitespace formatting (we have a bot that does that), or automated mass updates (like running a Resharper code cleanup on the entire codebase). 

(Also, I reserve the right to come up with other types of PRs that we don't like/want later on; this comment is not a binding contract. And heavens help you if you submit a PR that converts tabs->spaces or vice-versa.)
